### PR TITLE
22w46a atlases

### PR DESCRIPTION
**Pull request by YanisBft**

### CODE REVIEW

File: mappings/net/minecraft/client/texture/NativeImage.mapping [hunk 1]
```diff
@@ -198,6 +198,8 @@ CLASS net/minecraft/class_1011 net/minecraft/client/texture/NativeImage
 	METHOD method_4327 loadFromTextureImage (IZ)V
 		ARG 1 level
 		ARG 2 removeAlpha
+	METHOD method_47594 copyRect (Lnet/minecraft/class_1011;IIIIIIZZ)V
+		ARG 1 image
```
Comment (apple502j) [inline mappings/net/minecraft/client/texture/NativeImage.mapping:208]: This takes multiple arguments, could you map all?

Comment (YanisBft) [reply mappings/net/minecraft/client/texture/NativeImage.mapping:208]: Done!

**Review by apple502j — APPROVED** [review]

File: mappings/net/minecraft/client/texture/NativeImage.mapping [hunk 2]
```diff
@@ -198,6 +204,16 @@ CLASS net/minecraft/class_1011 net/minecraft/client/texture/NativeImage
...
+		ARG 2 x
+		ARG 3 y
+		ARG 4 translateX
+		ARG 5 translateY
```
Comment (enbrain) [inline mappings/net/minecraft/client/texture/NativeImage.mapping]: These are destination coordinates, not the difference between the original position and the target position.

**Review by enbrain — CHANGES_REQUESTED** [review]

File: mappings/net/minecraft/client/texture/NativeImage.mapping (hunk 2 above)
Comment (YanisBft) [reply mappings/net/minecraft/client/texture/NativeImage.mapping]: Fixed

**Review by enbrain — APPROVED** [review]

**Review by apple502j — APPROVED** [review]

**Review by modmuss50 — APPROVED** [review]

**Pull request merged**